src/e3/anod/loader.py

```diff
@@ -268,38 +273,25 @@ def load(self, repository: AnodSpecRepository) -> Callable[..., Anod]:
         raise SandBoxError(f"cannot find Anod subclass in {self.path}", "load")
 
 
-fixed_spec_repository: AnodSpecRepository | None = None
-# A local cache for the spec_repository.
-# This is set by set_spec_repository(), and used by spec().
-
-
-def set_spec_repository(spec_repository: AnodSpecRepository) -> None:
-    """Set the spec repository to use within the lifetime of this process.
-
-    This can be used to set the spec repository to use when calling spec()
-    below, without having to inspect the stack every time, which can be
-    expensive.
-
-    :param spec_repository: the spec repository to set
-    """
-    global fixed_spec_repository
-    fixed_spec_repository = spec_repository
-
-
 def spec(name: str) -> Callable[..., Anod]:
     """Load an Anod spec class.
 
+    Obsolete: keep until all from e3.anod.loader import spec are removed from the
```

Comment:
I think it is worth bumping the API version and rejecting calls to spec with the new version.